refactor(transform): Remove reflection_entry_points parameter

kegluneq opened the pull request:
Remove the now unnecessary `reflection_entry_points` parameter from the
Angular 2 transformer.

Support glob syntax for `entry_points`.

Commit:
refactor(transform): Remove reflection_entry_points parameter

Remove the now unnecessary `reflection_entry_points` parameter from the
Angular 2 transformer.

Support glob syntax for `entry_points`.

yjbanov commented on modules_dart/transform/lib/src/transform/common/options_reader.dart:
```diff
@@ -132,3 +130,10 @@ const CUSTOM_ANNOTATIONS_ERROR = '''
           import: ...
           superClass: ...''';
+
+void _warnDeprecated(Map config) {
```
Let's log an issue to remove this, say, a couple of weeks from now?

kegluneq replied:
Done, https://github.com/angular/angular/issues/3856, & set a reminder for myself the following Monday.

mary-poppins commented:
Merging PR #3854 on behalf of @yjbanov to branch [presubmit-yjbanov-pr-3854](https://github.com/angular/angular/tree/presubmit-yjbanov-pr-3854).

Commit:
fix(core): Fix type error

Ensure that values passed to `DomRenderer#setElementAttribute` are
strings. Currently, booleans can be passed to this method, resulting in
failures when running in Dart checked mode.

kegluneq commented:
Closed by 6c3c6060a58761734abb7304b044fb87f8deeb20
